STEERING-COMMITTEE.md

```diff
@@ -5,40 +5,52 @@ The Istio Steering Committee was formed to oversee the administrative aspects of
 ## Steering Committee Charter
 
 1. Define, evolve, and defend the vision, values, mission, and scope of the project - to establish and maintain the soul of Istio
-2. Charter and refine policy for community groups (including Working Groups and Committees)
-3. Define and evolve project governance structures and policies, including how contributors become committers/maintainers, approvers, reviewers, members, code of conduct, etc.
-4. Control access to, establish processes regarding, and provide a final escalation path for any Istio repository
-5. Control and delegate access to and establish processes regarding other project resources/assets, including artifact repositories, build and test infrastructure, web sites and their domains, blogs, social-media accounts, etc.
-6. Manage the Istio brand to decide which things can be called “Istio” and how that mark can be used in relation to other efforts or vendors.
-7. Resolve any dispute from the Technical Oversight Committee.
+2. Define and evolve project governance structures and policies, including how contributors become committers/maintainers, approvers, reviewers, members, code of conduct, etc.
```

Comment:
really hate to nitpick, but governance docs are a tricky thing and will be interpreted for years to come.  would recommend expanding etc in some way.  Perhaps "and any other topics the steering committee deems relevant."  That could probably be reduced in scope a bit - just a suggestion :)

Reply:
We did take much of the Istio Steering Committee charter from the K8s Steering Committee charter, so it has been tested a bit. With regards to #2, since that line is specifically speaking to project governance and Steering will have oversight of all governance questions, I think it's OK to leave it open.